contracts/Strategy.sol

```diff
@@ -112,17 +124,26 @@ contract Strategy is AccessControl, ERC20Rewards, StrategyMigrator { // TODO: I'
         isState(State.DEPLOYED)
```

Comment:
I see the `isState(State.DEPLOYED)` check has been added to the internal `_init()` should we remove it here?

Reply:
Removed, thanks